components/constellation/constellation.rs

```diff
@@ -1076,6 +1077,9 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             FromCompositorMsg::SetCursor(cursor) => {
                 self.handle_set_cursor_msg(cursor)
             }
+            FromCompositorMsg::OpenFileSelectDialog(patterns, multiple_files, sender) => {
```

Comment:
Having to use `FromCompositorMsg` is unfortunate. Maybe a `Request::Script` might be better? Or even its own type of request.